Source/Moya+Defaults.swift

```diff
@@ -3,8 +3,12 @@ import Alamofire
 /// These functions are default mappings to `MoyaProvider`'s properties: endpoints, requests, manager, etc.
 public extension MoyaProvider {
     public final class func DefaultEndpointMapping(target: Target) -> Endpoint<Target> {
-        let url = target.baseURL.URLByAppendingPathComponent(target.path).absoluteString
-        return Endpoint(URL: url, sampleResponseClosure: {.NetworkResponse(200, target.sampleData)}, method: target.method, parameters: target.parameters)
+        if let url = target.baseURL.URLByAppendingPathComponent(target.path) {
```

Comment:
Yes. But do you like to use this?
I don't like use "?".
In this case i prefer:
```
        if let url = target.baseURL.URLByAppendingPathComponent(target.path) {
            if let stringURL = url.absoluteString {
                return Endpoint(URL: url!, sampleResponseClosure: {.NetworkResponse(200, target.sampleData)}, method: target.method, parameters: target.parameters)
            }
        }
        fatalError("\(target) url is not valid")

```

What do you think?